lib/adapters/http/http.js

```diff
@@ -418,15 +447,15 @@ function HttpPouch(opts, callback) {
       callback = opts;
       opts = {};
     }
-    opts = utils.clone(opts);
-    if (opts.auto_encode === undefined) {
-      opts.auto_encode = true;
-    }
-    if (opts.auto_encode) {
-      docId = encodeDocId(docId);
-    }
-    opts.auto_encode = false;
-    api.get(docId + '/' + encodeAttachmentId(attachmentId), opts, callback);
```

Comment:
I wasn't a big fan of overloading `get()` here for both attachment-fetching and document-fetching. In my opinion, it's cleaner to  just directly call `ajax()`.